packages/babel-plugin-transform-unicode-property-regex/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@babel/plugin-transform-unicode-property-regex",
+  "version": "7.0.0-beta.3",
+  "description": "Compile Unicode property escapes in Unicode regular expressions to ES5.",
+  "homepage": "https://github.com/babel/babel/tree/master/packages/babel-plugin-transform-unicode-property-regex",
+  "main": "lib/index.js",
+  "engines": {
+    "node": ">=4"
+  },
+  "keywords": [
+    "babel-plugin",
+    "regex",
+    "regexp",
+    "regular expressions",
+    "unicode properties",
+    "unicode"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/babel/babel/tree/master/packages/babel-plugin-transform-unicode-property-regex.git"
```

Comment:
I *think* `.git` should be removed, since that url doesn't point to the git repo itself but to a subdirectory.